source/_integrations/group.markdown

```diff
@@ -18,24 +25,184 @@ ha_platforms:
   - notify
 ---
 
-Groups allow the user to combine multiple entities into one.
+The group integration lets you combine multiple entities into a single entity. Entities that are members of a group can be controlled and monitored as a whole.
 
-Check **Developer Tools** -> **States** and browse the **Current entities:** listing for all available entities.
+This can be useful for cases where you want to control, for example, the
+multiple bulbs in a light fixture as a single light in Home Assistant.
+
+Home Assistant can group multiple binary sensors, covers, fans, light, media players as a single entity, with the option of hiding the individual member entities.
+
+{% include integrations/config_flow.md %}
```

Comment:
Can we make a `helpers/config_flow.md` ? I want us to add a new helper My Button (but we didn't get to it yet). We can do this in another PR too.

Reply:
Let's do that for all the helpers when its set and ready to go :)